test/cmdlineargs.jl

```diff
@@ -28,6 +28,9 @@ let exename = `$(Base.julia_cmd()) --precompiled=yes --startup-file=no`
     @test success(`$exename -H $JULIA_HOME`)
     @test success(`$exename --home=$JULIA_HOME`)
 
+    # --config-gc-region-size
+    @test success(`$exename --config-gc-region-size=256M`)
```

Comment:
should read the value inside the julia process to verify that it was set correctly